spinn_machine/link.py

```diff
@@ -98,62 +81,13 @@ def destination_y(self):
         """
         return self._destination_y
 
-    @property
-    def multicast_default_from(self):
```

Comment:
Can this go? Well… it does only really seem to be used in `pacman.utilities.file_format_converters.create_file_constraints.CreateConstraintsToFile`…

Reply:
And SpiNNakerManchester/PACMAN#228 fixes that case. Good.